RoboFile.php

```diff
@@ -10,10 +10,16 @@ class RoboFile extends \Robo\Tasks
     /**
      * Run the Robo unit tests.
      */
-    public function test($args = "")
+    public function test($args = "", $options = ['coverage' => false])
     {
-        return $this->taskCodecept()
+        $taskCodecept = $this->taskCodecept()
             ->args($args);
+
+        if ($options['coverage']) {
+            $taskCodecept->coverageXml('build/logs/clover.xml');
```

Comment:
No change; I think it's probably some issue about relative paths. Not sure what the cwd is. Don't feel like setting up xdebug right now; could do that later to find out.